### PR TITLE
[MODULAR] Dauntless map tweaks and bounty pad

### PR DESCRIPTION
**Pull request by tmyqlfpir**
## About The Pull Request

This PR tweaks the following things for dauntless lava ruins:
* Removes improperly copied decals under hull tiles
* Adjusts the recreational area for the brig
* Adds many exterior cameras to the dauntless, and removes many interior ones (except for those in brig & interrogation)
* Adds a maxcap of 1MW to the powerator
* Adds a modified pirate bounty pad that allows the Dauntless crew to sell off goods for credits
* Adds a produce order console to the kitchen which is linked to the Interdyne budget account
* Adds a new area variable called `ignore_weather_sfx` which bypasses weather sfx (this is applied to the Dauntless)

## Why It's Good For The Game

This PR removes the possibility for the AI to go snooping in the Dauntless super sekret base, and can only view the exterior. It also tweaks areas such as dorms and brig to be easier on both the eyes and for movement.

The new bounty pad will allow players to use alterative forms of money, be it selling off the ore from netrunner gaming den, or other making goods for credits. This means they are not relying on using the powerator (which has been capped at 1MW).

## Proof Of Testing

I have ran this map on my local server and confirm it works.

The bounty pad works and I have ensured that there are enough sanity checks for it to not crash or break anything. It should not affect on station cargo prices.

https://github.com/Bubberstation/Bubberstation/assets/80724828/f8b5210a-4664-4c4b-8c6e-be17385f41c2

## Changelog

:cl:
add: Dauntless - Add exterior cameras to dauntless, removed many interior ones
fix: Dauntless - Removed incorrectly placed decals under hull objects and tweaked brig/dorms
balance: Interdyne powerator is now maxcapped to 1MW
add: Dauntless - Added new bounty pad and bounty console
add: Dauntless - Added new produce order console
/:cl:

### CODE REVIEW

**Comment by Mewchild** [comment]
I can't support the pAI being in the prison, as it gives access to both PDA messaging, comms, and a doorjack.

**Comment by shellspeed1** [comment]
One request from me, could we look at a produce console?

**Comment by shellspeed1** [comment]
Pardon, its specifically the service order console, I'll throw a screenshot up in a bit here.

**Comment by shellspeed1** [comment]
Tram Station Example

File: modular_zubbers/code/game/machinery/computer/orders/order_computer/cook_order_interdyne.dm [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+
+/obj/machinery/computer/order_console/cook/interdyne
+	name = "interdyne produce orders console"
+	desc = "An interface for ordering fresh produce and other. The cornerstone of any nutritious meal for the Syndicate crew."
+	circuit = /obj/item/circuitboard/computer/order_console/interdyne
+	order_categories = list(
+		CATEGORY_FRUITS_VEGGIES,
+		CATEGORY_MILK_EGGS,
+		CATEGORY_SAUCES_REAGENTS,
+	)
```
Comment (Majkl-J) [inline modular_zubbers/code/game/machinery/computer/orders/order_computer/cook_order_interdyne.dm:23]: This is inherited from parent iirc

```suggestion
```

File: modular_zubbers/code/game/machinery/computer/orders/order_computer/cook_order_interdyne.dm [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+	synced_bank_account = SSeconomy.get_dep_account(credits_account == "" ? ACCOUNT_CAR : credits_account)
+
+/obj/machinery/computer/order_console/cook/interdyne/ui_data(mob/user)
+	var/list/data = ..()
+	data["points"] = !synced_bank_account ? 0 : synced_bank_account.account_balance
+	return data
+
+/**
+ * This partically rewrites base behavior for express orders, and to subtract from department account
+ */
```
Comment (Majkl-J) [inline modular_zubbers/code/game/machinery/computer/orders/order_computer/cook_order_interdyne.dm:43]: So I am curious, would just giving all of the crew links to the account on their IDs be easier than rewriting all this? I suggest you give it a shot, could make a lot of this code more elegant.

Probably has to be showed into a post_equip on the loadouts.

**Review by Majkl-J — CHANGES_REQUESTED** [review]

File: modular_zubbers/code/game/machinery/computer/orders/order_computer/cook_order_interdyne.dm (hunk 2 above)
Comment (Majkl-J) [reply modular_zubbers/code/game/machinery/computer/orders/order_computer/cook_order_interdyne.dm:38]: > Perhaps I should default back to the current ID holder's account instead.

We could provide the admiral or whatever the highest rank is, with a card linked to the account so they can sort out payments to the crew if we do this.

**Review by Majkl-J — APPROVED** [review]

**Pull request merged**